config/jobs/kubernetes-sigs/cluster-api-provider-openstack/cluster-api-provider-openstack-presubmits.yaml

```diff
@@ -2,36 +2,127 @@ presubmits:
   kubernetes-sigs/cluster-api-provider-openstack:
   - name: pull-cluster-api-provider-openstack-build
     always_run: true
+    optional: false
     decorate: true
-    path_alias: sigs.k8s.io/cluster-api-provider-openstack
+    path_alias: "sigs.k8s.io/cluster-api-provider-openstack"
     labels:
-      preset-service-account: "true"
       preset-dind-enabled: "true"
     spec:
       containers:
-      - command:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210226-c001921-master
```

Comment:
just curious, do we want to get a released version?

Reply:
I think they only have versions like this and a job which automatically updates the used image for all ProwJobs every week or so (I just copied the image)